configs/config.annotated.yaml

```diff
@@ -0,0 +1,216 @@
+
+## Custom webhooks can be created and put in src/iris/webhooks.
+## Refer to their class name in this array.
+## The webhook will be available at /v0/webhooks/$name and requires being called with
+## ?application=xxx&key=xxx
+#webhooks: []
+
+#####################
+### Iris-sender
+#####################
+sender:
+  debug: True # Set to True, sender will not actually send messages
+  process_title: iris-sender
+  host: 127.0.0.1
+  port: 2321
+
+  ## Number of gevent workers per mode we send
+  workers_per_mode:
+    email: 500
+    #hipchat: 10
+
+  ## Default sender to reach out to if we can't determine one programmatically
+  master_sender:
+    host: 127.0.0.1
+    port: 2321
+
+  api_host: http://localhost:16649
+
+  ## App used for sending notification messages + incidents
+  sender_app: iris
+  ## RPC access log
+  access_log:
+    maxBytes: 10485760
+    filename: './logs/sender_rpc.access.log'
+
+  ## Optionally, use zookeeper for sender high availability.
+  #zookeeper_cluster: localhost:2181
+
+  ## If you're not using zookeeper, the following determines
+  ## whether this sender is a master and you hard code in
+  ## slaves to be cycled through
+  is_master: True
+
+#slaves:
+#  - host: 127.0.0.1
+#    port: 2322
+#  - host: 127.0.0.1
+#    port: 2323
+
+sender_workers: 1000
+
+## Enabled modules for resolving complex user roles to individual usernames, processed
+## in the order listed here.
+role_lookups:
+  - user # support for 'role' being username
+  - mailing_list # mailing list config; see bottom of file
+  #- oncall # oncall service integration
+  - dummy # dummy role lookup used for testing purposes. Disable this in production.
+
+#oncall-api: https://oncall.dummy.com  # oncall url
+
+## TODO
+applications:
+ - dummy_app
+
+## Vendors provides a way to add support for new notification mechanisms
+vendors: []
+#- type: iris_slack
+#  name: slack
+#  auth_token: ''
+#  base_url: 'https://slack.com/api/chat.postMessage'
+#  iris_incident_url: ''
+#  proxy: *proxy_shared
+#  message_attachments:
+#    fallback: 'Iris Alert Fired!'
+#    color: 'danger'
+#    pretext: '<!here> _Iris_ _Alert!_'
+
+#- type: iris_twilio
+#  name: twilio_1
+#  account_sid: ''
+#  auth_token: ''
+#  twilio_number: ''
+#  relay_base_url: ''
+
+## Hipchat support requires adding a new mode called "hipchat", in iris' mode table
+## Also, uncomment the hipchat workers to the sender['workers_per_mode'] hash above
+## Messages will be sent to $room_id with $username getting a mention.
+#- type: iris_hipchat
+#  name: hipchat
+#  auth_token: 'mytoken'
+#  debug: False
+#  base_url: 'https://api.hipchat.com'
+#  room_id: 1234567
+
+## Metrics
+metrics: dummy # output metrics to logs
+#metrics: prometheus
+#prometheus:
+#  iris-sync-targets:
+#    server_port: 8001
+#  iris-sender:
+#    server_port: 8002
+#
+#
+#metrics: influx
+#influxdb:
+#  connect:
+#    host: localhost
+#    port: 8086
+#    database: iris
+
+enable_gmail_oneclick: True
```

Comment:
Please change this default from `True` to `False`, and add a comment linking to https://developers.google.com/gmail/markup/reference/one-click-action